fixtures/artifactor_plugin.py

```diff
@@ -249,14 +249,13 @@ def pytest_unconfigure(config):
 def shutdown(config):
     with lock:
         proc = config._art_proc
-        if proc:
+        if proc.returncode is None:
```

Comment:
this holding true implies the code below is not correctly indented,

note that _art_proc can be None